playbot/java/src/main/java/com/PlayBot.java

```diff
@@ -0,0 +1,116 @@
+package com;
+
+import com.communication.BotCommunication;
+import com.communication.BotCommunicationImp;
+import com.enums.Move;
+import lombok.extern.log4j.Log4j2;
+import org.json.JSONObject;
+
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.util.Random;
+
+@Log4j2
+public class PlayBot {
```

Comment:
I am not sure it is a time... but we need to document this codes.

Reply:
Will do after we finish beta test and have nearly final version :)